src/assets/stylesheets/support.scss

```diff
@@ -0,0 +1,47 @@
+:local(.support-main) {
```

Comment:
Do you want to use any shared styles or variables here, or will that fail in some browsers?

Reply:
Yeah, just trying to minimize the surface area that could cause this to break again.